src/RenderWebGL.js

```diff
@@ -244,7 +258,12 @@ class RenderWebGL extends EventEmitter {
      * @param {number} blue The blue component for the background.
      */
     setBackgroundColor (red, green, blue) {
-        this._backgroundColor = [red, green, blue, 1];
+        this._backgroundColor4f = [red, green, blue, 1];
```

Comment:
I think this might generate less GC work:
```suggestion
        this._backgroundColor4f[0] = red;
        this._backgroundColor4f[1] = green;
        this._backgroundColor4f[2] = blue;
        this._backgroundColor4f[3] = 1;
```